packages/webdriverio/src/commands/element/isExisting.js

```diff
@@ -41,5 +41,7 @@
  */
 
 export default function isExisting () {
```

Comment:
Is it worth to cover other `isSomething` methods?

Reply:
That's a good point, I'll take a look at how these methods and adjust them if needed.

Reply:
The only other command I could find using this was `isDisplayed`. It has been adjusted in a50e10d .